### PR TITLE
Add SymbolOptions for CppVsDbg

### PR DESCRIPTION
**Pull request by WardenGnaw**
Add schema to allow customization of how the debugger searches for symbols.

With `"searchMicrosoftSymbolServer": true`, engine logs show:

```
<- (E) {"seq":108,"type":"event","event":"progressUpdate","body":{"progressId":"1","message":"Loading symbols for KernelBase.dll","percentage":0}}
<- (E) {"seq":110,"type":"event","event":"progressUpdate","body":{"progressId":"1","message":"Loading symbols for KernelBase.dll from: D:\\Projects\\vcpp\\ConsoleApplication1\\Debug","percentage":0}}
<- (E) {"seq":112,"type":"event","event":"progressUpdate","body":{"progressId":"1","message":"Loading symbols for KernelBase.dll from: C:\\Windows\\SysWOW64\\","percentage":0}}
<- (E) {"seq":114,"type":"event","event":"progressUpdate","body":{"progressId":"1","message":"Loading symbols for KernelBase.dll from: C:\\WINDOWS\\","percentage":0}}
<- (E) {"seq":116,"type":"event","event":"progressUpdate","body":{"progressId":"1","message":"Loading symbols for KernelBase.dll from: C:\\WINDOWS\\symbols\\dll\\","percentage":0}}
<- (E) {"seq":118,"type":"event","event":"progressUpdate","body":{"progressId":"1","message":"Loading symbols for KernelBase.dll from: C:\\WINDOWS\\dll\\","percentage":0}}
<- (E) {"seq":120,"type":"event","event":"progressUpdate","body":{"progressId":"1","message":"Loading symbols for KernelBase.dll from: C:\\Users\\USER\\AppData\\Local\\Temp\\symcache\\wkernelbase.pdb\\cbe4700ef26cd25c4b4801e9e1d2e53e1\\","percentage":0}}
```

**Tested:**
```
"moduleFilter": {
   "mode": "loadAllButExcluded",
   "excludedModules": [ "KernelBase.dll" ]
}
```
Logs;
```
<- (E) {"seq":81,"type":"event","event":"progressUpdate","body":{"progressId":"1","message":"Loading symbols for KernelBase.dll","percentage":0}}
<- (E) {"seq":83,"type":"event","event":"output","body":{"category":"console","output":"Loaded 'C:\\Windows\\SysWOW64\\KernelBase.dll'. Symbol loading disabled by Include/Exclude setting.\n"}}

```

@gregg-miskelly

### CODE REVIEW

File: Extension/package.json [hunk 1]
```diff
@@ -2245,6 +2245,83 @@
...
+                        "type": "string",
+                        "enum": [
+                          "loadAllButExcluded",
+                          "loadOnlyIncluded"
+                        ],
+                        "enumDescriptions": [
+                          "%c_cpp.debuggers.VSSymbolOptionsModuleFilter.mode.loadAllButExcluded.enumDescriptions%",
+                          "%c_cpp.debuggers.VSSymbolOptionsModuleFilter.mode.loadOnlyIncluded.enumDescriptions%"
+                        ],
+                        "description": "Controls which of the two basic operating modes the module filter operates in.",
```
Comment (gregg-miskelly) [inline Extension/package.json]: >Controls which of the two basic operating modes the module filter operates in. [](http://example.com/codeflow?start=40&length=78)

Should this be localized?

**Review by gregg-miskelly — APPROVED** [review]
Otherwise LGTM

**Comment by sean-mcmanus** [comment]
Did you want to get this into 1.5.0-insiders?

**Comment by WardenGnaw** [comment]
> Did you want to get this into 1.5.0-insiders?

This is just a schema change, so it doesn't really matter. The configuration options are already consumed in the 1.4.1 CppVsdbg debugger.

**Review by calgagi — APPROVED** [review]
LGTM

**Review by gregg-miskelly — APPROVED** [review]

**Review by mustafahemidaa — COMMENTED** [review]
f6692089a415fcd8f05ff689d9eb262b6bb8e3ca

**Pull request merged**